bundles/org.openhab.binding.tibber/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -115,4 +116,10 @@
 		<description>Accumulated Production since Midnight</description>
 		<state pattern="%.3f %unit%"></state>
 	</channel-type>
+	<channel-type id="tomorrow_prices">
```

Comment:
Consider if this channel should be marked as advanced, since it contains a JSON string that would need to be parsed by a rule:
```suggestion
	<channel-type id="tomorrow_prices" advanced="true">
```

Reply:
@seime - this is the only remaining comment to be considered before merging, now wondering if you might have missed it?

Reply:
I most certainly have, will fix tonight :)